shtuff.py

```diff
@@ -111,11 +112,17 @@ def get_cmd_file(pid):
     return data_dir(f"{pid}.command")
 
 def find_nearest_shtuff_process():
-    return subprocess.run(
-        """pstree -lps $$ | grep -Eo "shtuff\([0-9]+\)" | grep -Eo '[0-9]+'""",
-        shell=True,
-        capture_output=True,
-    ).stdout.decode().strip()
+    def ppid(process):
+        parent = process.parent()
+        if parent is None:
+            return
```

Comment:
This works, buts "more pythonic" to explicitly return None instead. Plain return is just for functions that don't return anything (as in, people should never use their return value)